python/python-core/meet-python/why-learn-python.md

```diff
@@ -34,9 +34,7 @@ Python is seen as the number one language to learn. Here are a few reasons why:
 
 This makes it an *ideal language for beginners to learn*. And it acts as a stepping-stone for learning other object-oriented [1] languages.
 
-Python is also concise.
-
-For the same task, Python requires *3-5* times less code than Java, and up to *10* times less code than C++.
+Python is also concise and usually requires less code for the same task than languages such as Java or C++.
```

Comment:
let's replace "such as" with "like". I've also added "other" here to keep the same message
```suggestion
Python is also concise and usually requires less code for the same task than other languages like Java or C++.
```